src/integration-tests/test_admin_client.py

```diff
@@ -2,22 +2,74 @@
 This suite of test cases exercises admin session connection and some basic
 commands.
 """
-
+import dataclasses
 import json
 import re
+from typing import Dict, Optional, Tuple
 
 from blazingmq.dev.it.fixtures import Cluster, single_node, order  # pylint: disable=unused-import
 from blazingmq.dev.it.process.admin import AdminClient
+from blazingmq.dev.it.process.client import Client
+import blazingmq.dev.it.testconstants as tc
 
 
-def test_admin(single_node: Cluster):
-    cluster: Cluster = single_node
+def get_endpoint(cluster: Cluster) -> Tuple[str, int]:
     endpoint: str = cluster.config.definition.nodes[0].transport.tcp.endpoint  # type: ignore
 
     # Extract the (host, port) pair from the config
     m = re.match(r".+://(.+):(\d+)", endpoint)  # tcp://host:port
     assert m is not None
-    host, port = str(m.group(1)), int(m.group(2))
+
+    return str(m.group(1)), int(m.group(2))
+
+
+@dataclasses.dataclass
+class PostRecord:
+    domain: str
+    queue_name: str
+    num: int
+
+    @property
+    def uri(self) -> str:
+        return f"bmq://{self.domain}/{self.queue_name}"
+
+    def append(self, other: "PostRecord") -> None:
+        assert self.domain == other.domain
+        assert self.queue_name == other.queue_name
+        self.num += other.num
+
+
+def post_n_msgs(producer: Client, task: PostRecord, posted: Optional[Dict[str, PostRecord]] = None) -> None:
+    """
+    Open a connection for the specified 'producer' to the specified 'uri'
```

Comment:
The function arguments are `[producer, task, posted]`, but the documentation mentions `[producer, uri, n]`. Please fix the docs.